docs/source/get_started/new_project.md

```diff
@@ -42,7 +42,14 @@ You can add one or more of the options, or follow the default and add none at al
 
 ### Project examples
 
-TO DO
+The CLI offers the option to include example pipelines. Your choice of tools determines which spaceflights example starters are provided. Here's a guide to understanding which starter examples are used based on your selections:
```

Comment:
It sounds like you would get multiple examples, but it would just be one example you get: 
```suggestion
The CLI offers the option to include example pipelines. Your choice of tools determines which spaceflights starter example is provided. Here's a guide to understanding which starter examples are used based on your selections:
```